src/test/resources/trywithresources/input/com/example/Simple.java

```diff
@@ -0,0 +1,23 @@
+package com.example;
+
+import org.example.Resource;
+import org.example.OtherResource;
+import org.example.ThirdResource;
+
+class Simple {
+
+    private ThirdResource r;
```

Comment:
For this test case, we might want to change this to `private final ThirdResource r = new ThirdResource()` to prevent the compilation error we see in the CI. Not sure how Specimin will handle this (if it will remove the initializer or not) but we could force Specimin to preserve all used fields if they are used with try-with-resources.

Reply:
I changed it to just initialize it to null, which is what Specimin would do anyway. The actual value of the resource doesn't matter here, since we're not going to run the code - javac just checks that it's final.